.github/workflows/test.yml

```diff
@@ -9,10 +9,29 @@ jobs:
     runs-on: ${{ matrix.platform }}
     steps:
     - name: Install Go
-      uses: actions/setup-go@v1
+      uses: actions/setup-go@v2
       with:
         go-version: ${{ matrix.go-version }}
     - name: Checkout code
       uses: actions/checkout@v2
     - name: Test
-      run: go test ./...
+      run: |
+        go version
+        go test ./...
+
+  test-gotip:
+    runs-on: ubuntu-latest
```

Comment:
Re docker - there's no need, because Actions already has a recent enough version of Go pre-installed. Adding containers here would just make the job more complex, I think, because we would probably have to do everything within a single container.

> just go get gotip

Assuming you mean https://godoc.org/golang.org/dl/gotip, I could use it, but that's really just a convenience. For our limited use case, four shell lines are just as simple as two cmd/go lines, and they're faster as we skip the middleman.

Reply:
Makes sense 👍